src/components/organisms/carousel_js/EmblaCarousel.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { useEffect } from 'react'
+import useEmblaCarousel from 'embla-carousel-react'
+import Autoplay from 'embla-carousel-autoplay'
+import '../carousel_js/embla.css'
+
+import background1 from 'src/assets/images/backgrounds/1.gif'
+import background2 from 'src/assets/images/backgrounds/2.gif'
+import background3 from 'src/assets/images/backgrounds/3.gif'
+
+
+export function EmblaCarousel() {
+  const [emblaRef] = useEmblaCarousel({ loop: false }, [Autoplay()])
+  const placeholderImages =["src/assets/images/backgrounds/1.gif", "src/assets/images/backgrounds/2.gif", "src/assets/images/backgrounds/3.gif"]
```

Comment:
you should use the backgrounds imported above

you can do `background1.src, background2.src, ..` and so on